utils/xml/agent-common.xsd

```diff
@@ -1,454 +1,465 @@
-      If 0 then this value is automatically computed from the number of available cpus.
-    </xsd:documentation></xsd:annotation>
-    <xsd:restriction base="xsd:integer">
-      <xsd:minInclusive value="0"/>    
-    </xsd:restriction>
-  </xsd:simpleType>
-
-  <xsd:simpleType name="ConfNbWorkersType">
-    <xsd:annotation><xsd:documentation xml:lang="en">
-      The number of ProActive workers to start per runtime (1 by default).
-
-      If present but empty then the node is started with "-w", which means that it will use as many workers as there are cores.
-    </xsd:documentation></xsd:annotation>
-    <xsd:union memberTypes="xsd:positiveInteger">
-      <xsd:simpleType>
-        <xsd:restriction base="xsd:string">
-          <xsd:enumeration value=""/>
-        </xsd:restriction>
-      </xsd:simpleType>
-    </xsd:union>
-  </xsd:simpleType>
-      
-  <xsd:simpleType name="ConfProtocolType">
-    <xsd:annotation><xsd:documentation xml:lang="en">
-    The communication protocol to be used by the started ProActive runtime     
-    </xsd:documentation></xsd:annotation>
-    <xsd:list itemType="nonEmptyString"/>
-  </xsd:simpleType>
-
-  <xsd:complexType name="ConfPortRangeType">
-    <xsd:annotation><xsd:documentation xml:lang="en">
-      A range of the TCP ports to be used by the ProActive runtimes.
-      
-      The number of ports in the range must be greater than the number of runtimes.
-    </xsd:documentation></xsd:annotation>
-    <xsd:attribute use="required" type="portType" name="first"/>
-    <xsd:attribute use="required" type="portType" name="last"/>
-  </xsd:complexType>
-
-  <xsd:simpleType name="ConfOnRuntimeExitScriptType"> 
-    <xsd:annotation><xsd:documentation xml:lang="en">
-    The path to an executable to be started each time a forked ProActive runtime exits. It can be used to cleanup some ressources, or to gather some results
-    </xsd:documentation></xsd:annotation>
-    <xsd:list itemType="nonEmptyString"/>
-  </xsd:simpleType>
-
-
-
-
-
-  <xsd:simpleType name="javaClassName" >
-    <xsd:restriction base="xsd:string">
-      <xsd:pattern value="([\w]+\.)*[\w]+"/>
-    </xsd:restriction>
-  </xsd:simpleType>
-
-  <xsd:simpleType name="nonEmptyString">
-    <xsd:restriction base="xsd:string">
-      <xsd:minLength value="1" />
-      <xsd:pattern value=".*[^\s].*" />
-    </xsd:restriction>
-  </xsd:simpleType>
-
-  <xsd:simpleType name="autoType">
-    <xsd:restriction base="xsd:string">
-      <xsd:enumeration value="auto"/>
-    </xsd:restriction>
-  </xsd:simpleType>
-  
-  <xsd:simpleType name="portType">
-    <xsd:restriction base="xsd:unsignedShort">
-      <xsd:minInclusive value="0"/>
-      <xsd:maxExclusive value="65535"/>
-    </xsd:restriction>
-  </xsd:simpleType>
-</xsd:schema>
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema
+        xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+        elementFormDefault="qualified">
+
+
+  <!--
+                 /!\   PLEASE DO NOT REFORMAT THIS FILE /!\
```

Comment:
Please do not reformat this file ;-)
Don't know what we are supposed to do here, but it is indeed difficult to see the changes after reformatting

Reply:
I used the same file as agent-linux to be sure that they both use the same very file as intended, I believe this is the way to go, but if you want I can do the change locally